recipes/numpy/meta.yaml

```diff
@@ -0,0 +1,49 @@
+{% set version="1.11.0" %}
+
+package:
+  name: numpy
+  version: {{ version }}
+
+source:
+  fn: numpy-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/n/numpy/numpy-{{ version }}.tar.gz
+  md5: bc56fb9fc2895aa4961802ffbdb31d0b
+  # patches:
+    # - osx-dist.patch          # [osx]
+
+build:
+  number: 0
+  features:
+    - blas_openblas
+
+requirements:
+  build:
+    - cython
+    - gcc
```

Comment:
So, even though NumPy has `f2py`, it shouldn't actually compile any Fortran code. There is some for testing, but I don't believe this is run as part of the standard build process. Thus, I think we can drop this. We should try doing that after our `blas` variant package is out.